### PR TITLE
feat(templateManager): add getEmbeddedView tests, fix selector

### PR DESCRIPTION
**Pull request by BioPhoton**

### CODE REVIEW

File: libs/template/src/lib/core/utils/template-manager_creator.ts [hunk 1]
```diff
@@ -86,6 +95,14 @@ export function createTemplateManager<C extends object, N extends string = strin
       }
     },
 
+    getEmbeddedView(name: N): EmbeddedViewRef<C> {
+      if (viewCache.has(name)) {
+        return viewCache.get(name);
+      } else {
+        return undefined;
```
Comment (kajetansw) [inline libs/template/src/lib/core/utils/template-manager_creator.ts:102]: Nitpick: `Map.prototype.get()` seems to return `undefined` by itself when no value is provided for a given key.

Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/get

File: libs/template/src/lib/core/utils/template-manager_creator.ts [hunk 2]
```diff
@@ -19,6 +19,15 @@ export interface TemplateManager<C extends object, N extends string = string> {
    */
   addTemplateRef(name: N, templateRef: TemplateRef<C>): void;
 
+  /**
+   * @description
+   * Returns a template from the internal templateRefCache map.
+   *
+   * @param name
+   * @param templateRef
```
Comment (kajetansw) [inline libs/template/src/lib/core/utils/template-manager_creator.ts:27]: This param does not exist.

**Comment by kajetansw** [comment]
Aside from two things above - looks fine by me 👍

File: libs/template/src/lib/core/utils/template-manager_creator.ts [hunk 3]
```diff
@@ -86,6 +95,14 @@ export function createTemplateManager<C extends object, N extends string = strin
       }
     },
 
+    getEmbeddedView(name: N): EmbeddedViewRef<C> {
+      if (viewCache.has(name)) {
+        return viewCache.get(name);
+      } else {
+        return undefined;
+      }
```
Comment (BioPhoton) [inline libs/template/src/lib/core/utils/template-manager_creator.ts:103]:
```suggestion
 return viewCache.get(name);
```

Comment (BioPhoton) [inline libs/template/src/lib/core/utils/template-manager_creator.ts]:
```suggestion
      return viewCache.get(name);
```

File: libs/template/src/lib/core/utils/template-manager_creator.ts (hunk 2 above)
Comment (BioPhoton) [inline libs/template/src/lib/core/utils/template-manager_creator.ts:27]:
```suggestion
```

**Comment by BioPhoton** [comment]
Unfortunately I can't apply my own change suggestions :D

**Pull request merged**